app/views/api/v3/analytics/user_analytics/_monthly_gender_table.html.erb

```diff
@@ -14,7 +14,10 @@
     <tr>
       <td class="row-label no-padding"><%= month_date.strftime("%B") %></td>
       <td>
-        <%= zero_if_unavailable(@user_analytics.stats_across_genders_for_month(resource, month_date)) %>
```

Comment:
We've refactored the data-structure to remove the need for the presenter method `stats_across_genders_for_month` entirely. It's a lot simpler this way.

Reply:
Yeah, we've exposed `follow_ups` under `grouped_by_date` which "un-segments" the gender-segmentation and pulls out [date-only segmentation](https://github.com/simpledotorg/simple-server/pull/888/files#diff-e9e2bbe1c12d40d2ea6f7a9d383a866aR145).

The mapping has also been inverted, previously we had a map that was like:

```ruby
{ male: { date_1: 2, date_2: 3} } 
```

Now we have, 

```ruby
{ [date, gender_1] => 3,
  [date_2, gender_1] => 4 } 
```

This can be used if you're showing gender + date split **and** just date-split by "reducing" over the genders.